src/blocks/ankitbajpai65/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div id="mainDiv">
+        <div class="footer">
+            <div class="footerDiv">
+                <div class="footerPart">
+                    <h3>CATEGORIES</h3>
+                    <ul>
+                        <li><a href="">Category1</a></li>
+                        <li><a href="">Category2</a></li>
+                        <li><a href="">Category3</a></li>
+                        <li><a href="">Category4</a></li>
+                    </ul>
+                </div>
+                <div class="footerPart">
+                    <h3>HELP</h3>
+                    <ul>
+                        <li><a href="">Help1</a></li>
+                        <li><a href="">Help2</a></li>
+                        <li><a href="">Help3</a></li>
+                        <li><a href="">Help4</a></li>
+                    </ul>
+                </div>
+                <div class="footerPart">
+                    <h3>GET IN TOUCH</h3>
+                    <p>Any questions? Let us know in store at BP 473 Complexe universitaire Al Qods, Oujda 60000 or call
+                        us
+                        on (+212) 65964665</p>
+                    <div class="imgDiv">
+                        <span><img src="icons\fb.png" alt=""></span>
```

Comment:
Please provide the license/source link for the Images to make sure that you have the right to use those.